community_images/yourls/official/overrides.yml

```diff
@@ -0,0 +1,18 @@
+image:
```

Comment:
if you dont have k8s runtime, why do you need overrides file

Reply:
Removing it right now.